content/sensu-go/6.2/observability-pipeline/observe-entities/entities.md

```diff
@@ -26,23 +26,34 @@ See [the announcement on our blog][11] for more information about our usage poli
 When an agent connects to a backend, the agent entity definition is created from the information in the `agent.yml` configuration file.
 The default `agent.yml` file location [depends on your operating system][35].
 
+### Manage agent entities via the backend
+
 You can manage agent entities via the backend with [sensuctl][37], the [entities API][36], and the [web UI][33], just like any other Sensu resource.
 This means you do not need to update the `agent.yml` configuration file to add, update, or delete agent entity attributes like subscriptions and labels.
+This is the default configuration for agent entities.
```

Comment:
I just took a guess that this configuration is still an option, and that it is the default.

Reply:
That's right; this new behavior is opt-in, and the default is still using the backend to manage entities.